src/compas_timber/design/workflow.py

```diff
@@ -164,10 +164,11 @@ def comply(self, beams, max_distance=1e-6):
             beams = list(beams)
             if beam_cats == set([self.category_a, self.category_b]):
                 solver = ConnectionSolver()
-                if (
-                    self.joint_type.SUPPORTED_TOPOLOGY
-                    == solver.find_topology(beams[0], beams[1], max_distance=max_distance)[0]
-                ):
+                found_topology = solver.find_topology(beams[0], beams[1], max_distance=max_distance)[0]
+                supported_topo = self.joint_type.SUPPORTED_TOPOLOGY
+                if not isinstance(supported_topo, list):
+                    supported_topo = [supported_topo]
+                if found_topology in supported_topo:
```

Comment:
we should eventually change all `SUPPORTED_TOPOLOGY` attributes so that the only check necessary is 
```python
if found_topology in self.joint_type.SUPPORTED_TOPOLOGY:
   ...
```
but we can do that in a separate PR

Reply:
I agree. But as you said, I thought that it would be better if it happened in a separate PR. I can create an issue in the backlog.